src/analysis/processing/qgsalgorithmextractspecificvertices.cpp

```diff
@@ -49,7 +49,7 @@ QString QgsExtractSpecificVerticesAlgorithm::groupId() const
 
 QString QgsExtractSpecificVerticesAlgorithm::shortHelpString() const
 {
-  return QObject::tr( "This algorithm takes a line or polygon layer and generates a point layer with points "
+  return QObject::tr( "This algorithm takes a vector layer and generates a point layer with points "
                       "representing specific vertices in the input lines or polygons. For instance, this algorithm "
```

Comment:
```suggestion
                      "representing specific vertices in the input features. For instance, this algorithm "
```
should we not include (multi)point here also?

Reply:
Maybe here it is better "geometry".